python/azureml/hybrid.py

```diff
@@ -0,0 +1,316 @@
+# ------------------------------------------------------------------------------------------------------
+# The file implemented three ways to merge a local model and a azure-proxy model for hybrid inferencing:
+# 1. Run either one.
```

Comment:
It'll really help if we can make some ASCII drawings showing what configurations we are building between the edge and proxy models.

Reply:
Will move this part to README.md